app/controllers/shopify_app/fulfillment_services_controller.rb

```diff
@@ -0,0 +1,23 @@
+module ShopifyApp
+  class FulfillmentServicesController < ApplicationController
+    class ShopifyApp::MissingFulfillmentServiceError < StandardError; end
+
+    def action_missing(name)
+      render json: fulfillment_service_klass.send(name, fulfilment_service_params)
+    end
```

Comment:
`action_missing` is basically `method_missing` in controllers. I'll change it to be more readable: 

```ruby
    def fetch_stock
      render json: fulfillment_service_klass.send(:fetch_stock, fulfilment_service_params)
    end
```